eve/main.yml

```diff
@@ -285,6 +286,134 @@ stages:
           doStepIf: false
           alwaysRun: true
 
+  single-node-upgrade-test:
+    worker:
+      type: openstack
+      image: CentOS-7-x86_64-GenericCloud-1809.qcow2
+      flavor: m1.large
+      path: eve/workers/openstack-single-node
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
+          name: Report IP and install SSH keys
+          command: >
+            ip a &&
+            mkdir -p ~/.ssh &&
+            echo "%(secret:ssh_pub_keys)s" >> ~/.ssh/authorized_keys
+      - SetPropertyFromCommand: &set_prev_version
+          name: Set previous version
+          property: metalk8s_prev_version
+          command: >
+            bash -c '
+              source VERSION &&
+              VERSION_MINOR=$((VERSION_MINOR-1))
+            echo $VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH$VERSION_SUFFIX'
+      - ShellCommand: &get_prev_iso
+          name: Get %(prop:metalk8s_prev_version)s image
+          command: >
+            isourl="/pre/metalk8s-%(prop:metalk8s_prev_version)s.iso";
+            curl -s -XGET -o metalk8s.iso
+            "%(prop:artifacts_private_url)s$isourl"
+          haltOnFailure: true
+      - ShellCommand: &get_prev_iso_checksum
+          name: Get %(prop:metalk8s_prev_version)s checksum
+          command: >
+            shasumurl="/pre/SHA256SUM-%(prop:metalk8s_prev_version)s";
+            curl -s -XGET -o SHA256SUM
+            "%(prop:artifacts_private_url)s$shasumurl"
+          haltOnFailure: true
+      - ShellCommand: &verify_prev_iso_checksum
+          name: Verify %(prop:metalk8s_prev_version)s checksum
+          command: sha256sum -c SHA256SUM
+          haltOnFailure: true
+      - ShellCommand: &get_iso
+          name: Get %(prop:metalk8s_version)s image
+          command: >
+            curl -s -XGET -o /tmp/metalk8s.iso
+            "%(prop:artifacts_private_url)s/metalk8s.iso"
+          haltOnFailure: true
+      - ShellCommand: &get_iso_checksum
+          name: Get %(prop:metalk8s_version)s checksum
+          command: >
+            curl -s -XGET -o /tmp/SHA256SUM
+            "%(prop:artifacts_private_url)s/SHA256SUM"
+          haltOnFailure: true
+      - ShellCommand: &verify_iso_checksum
```

Comment:
`&blabla` is special yaml Anchor (you can google it for more informations)
Here you redefine the `verify_iso_checksum`

You should remove all the content of hte ShellCommand and only keep 
```
- ShellCommand: *verify_iso_checksum
```

And btw it's the same for all other steps with `: &blabla`

And as suggest in downgrade PR you should add some for common upgrade/downgrade steps

Reply:
yes, we can do that. which ever PR gets merge first will re-use the shared tags. I got a green light from @sayf-eddine-scality  to go first if i didnt get a flaky.

Reply:
Ok so lets definie tag in your PR like I suggested here
https://github.com/scality/metalk8s/pull/1730/files#r326615160

Reply:
ok!